pkg/kubernetes/data_test.go

```diff
@@ -110,28 +110,6 @@ func testDataFlat() testData {
 	}
```

Comment:
I'd hope that we could ditch this file in favour of using the fixture files, as we have in `TestWalkJSON`. I didn't want to commit to restructuring the `Build` tests for now though, that can happen separately.

As an aside, you could argue that the tests for `Build` should entirely replace the tests we have for `walkJSON`, as `walkJSON` is called only by `Build` and is a private function. Not that fussed though.